packages/contentful/src/crn/queries/research-outputs.queries.ts

```diff
@@ -166,8 +166,8 @@ export const researchOutputContentQueryFragment = gql`
         }
       }
     }
-    linkedFrom {
-      researchOutputsCollection(limit: 20) {
+    linkedFrom @include(if: $fetchLinkedRelatedResearch) {
```

Comment:
I would add this same include filter to the relation in the other direction as well (at L151) so we always either get all of the related outputs, or none of them. Getting _some_ is liable to be a bit confusing.

Reply:
it makes sense! I've changed it here [condition to fetch related research added to relatedResearchCollection](https://github.com/yldio/asap-hub/pull/3474/commits/ba0c9a11533bc4b28d88331299b03ccc34ed52bc)